gmm/handler.go

```diff
@@ -1173,18 +1173,16 @@ func communicateWithUDM(ue *context.AmfUe, accessType models.AccessType) error {
 
 func getSubscribedNssai(ue *context.AmfUe) {
 	amfSelf := context.AMF_Self()
-	if ue.NudmSDMUri == "" {
```

Comment:
Given that you are removing this variable from here, would it be possible that you also remove from the other places? because I think it is actually used anywhere in the code (besides some "self-checks" as shown below). It can be done as part of this PR or in another PR.
```go
	ue.NudmSDMUri = sdmUri
	if ue.NudmUECMUri == "" || ue.NudmSDMUri == "" {
```

Reply:
@ghislainbourgeois, please let me know what you think about my previous comment. Thanks!

Reply:
I am not sure what the impact would be in removing it from the other places. I assume the URI of the UDM is saved in the UE context as a form of cache to make things faster, but I definitely do not have the whole context here.

I think I could propose a separate PR with this change, and it would make it safer to test.

If you have any input on running multiple UDMs, that would also be interesting, as I think this cache per UE would only really be useful in the case of multiple UDM instances.

What do you think?

Reply:
Sure, opening another PR for the removal of `NudmSDMUri` would be fine.
For the deployment of multiple UDMs, I think @thakurajayL would be the best person to with it.